_posts/2021-04-30-soliditylangorg-umbrella-domain.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+published: true
+title: 'What happened with Solidity-related Domains?'
+date: '2021-04-30'
+author: Franziska Heintel
+category: Explainers
+---
+
+Some time ago we decided to get a domain that the Solidity team has easy access to in
+order to streamline efforts and initiatives that were hosted on other domains before.
+
+And so, ``soliditylang.org`` was born! 🎉
+
+We announced most of these domain changes individually [on Twitter](https://twitter.com/solidity_lang/status/1329031139495964673?s=20), but we want to take a moment to
+also officially announce it here on the blog
+and explain the various subdomains we have now, what they are for and which older
+domains may be deprecated in future.
+
+In short, we currently maintain the following pages under the soliditylang.org umbrella:
+
+- Solidity Portal: [soliditylang.org](https://soliditylang.org/)
+- Documentation: [docs.soliditylang.org](https://docs.soliditylang.org/)
+- Blog: [blog.soliditylang.org](https://blog.soliditylang.org/)
+- Static Binaries: [binaries.soliditylang.org](https://binaries.soliditylang.org/)
+- Forum: [forum.soliditylang.org](https://forum.soliditylang.org/)
+- Solidity Summit: [summit.soliditylang.org](https://summit.soliditylang.org/)
+- Solidity Underhanded Contest: [underhanded.soliditylang.org](https://underhanded.soliditylang.org/)
+
+## Solidity Portal
+
+The Solidity language portal at [soliditylang.org](https://soliditylang.org/) is a one-stop-shop for all important links and high-level information around Solidity. It includes information on the latest Solidity release, how to get started, information on language design and features recent blog posts and news.
+## Documentation
+
+The Solidity documentation was originally available at ``solidity.readthedocs.io``. Now, the readthedocs domain redirects to the documentations' new home: [docs.soliditylang.org](https://docs.soliditylang.org/). Since it's a 301 redirect, all old links are still working and will redirect directly to the correct section in the docs on the new domain.
```

Comment:
```suggestion
The Solidity documentation was originally available at ``solidity.readthedocs.io``. Now, the readthedocs domain redirects to the documentations' new home: [docs.soliditylang.org](https://docs.soliditylang.org/). Since it's a permanent redirection ([HTTP 301](https://en.wikipedia.org/wiki/HTTP_301)), all old links are still working and will redirect directly to the correct section in the docs on the new domain.
```